### ISSUE TITLE
Inform users to select variations.

### ISSUE THREAD

**Issue by highacid** [woothemes/woocommerce#10909]
## EXPLANATION OF THE ISSUE

What happens, under which versions, under what conditions, when, and what were you expecting instead.

## STEPS TO REPRODUCE THE ISSUE

1. List steps to reproduce your issue so we can replicate.

## SYSTEM STATUS REPORT

```
Grab the system status report from WooCommerce > System Status and paste it here.
```

Hello,

We have a site with variable products, the issue is that when someone wants to add something in basket without selecting a variation, the button wont work, also there will be no notification from woocommerce to used to let him know to pick a size or so. Can this someone be solved?

Regards,
Cosmin

**Comment by mikejolley** [comment]
https://dl.dropboxusercontent.com/s/ehmpm7zlxuz20to/2016-05-12%20at%2014.33.png?dl=0

**Comment by WPJobs** [comment]
Hi guys

I have made a test installation where you can see that notification is not displayed when you click on "add to cart" button without selecting a variation.

Latest WooCommerce & WordPress, no any additional plugins, default WordPress theme, WooCommerce dummy data

test installation: http://goo.gl/XrXQ2N
system report: http://pastebin.com/gwRqCm5P

tested in Chrome 50 (Windows)

**Comment by mikejolley** [comment]
Maybe it's in master then, I forget. The button has disabled attribute so it's not clickable on your test site.

**Comment by WPJobs** [comment]
@mikejolley, thank you for a fast reply! So, in defaults, this message can not be displayed? "Disabled" attribute sets by JavaScript, i can't find it in WooCommerce template files to remove

**Comment by mikejolley** [comment]
I think that must have been something changed in 2.6 (master) branch then. Look in the new js files in master.

**Comment by WPJobs** [comment]
@mikejolley you are right, it seems this will be fixed in 2.6, thank you!
